### PR TITLE
Added genvalidity-hspec-binary

### PR DESCRIPTION
**Pull request by Nickske666**
Just added genvalidity-hspec-binary.

### CODE REVIEW

File: genvalidity-hspec-binary/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Tom Sydney Kerckhove
```
Comment (NorfairKing) [inline genvalidity-hspec-binary/LICENSE]: Copyright 2017

File: genvalidity-hspec-binary/genvalidity-hspec-binary.cabal [hunk 2]
```diff
@@ -0,0 +1,41 @@
+name:                genvalidity-hspec-binary
+version:             0.1.0.0
+-- synopsis:
+-- description:
+homepage:            https://github.com/githubuser/genvalidity-hspec-binary#readme
+license:             BSD3
```
Comment (NorfairKing) [inline genvalidity-hspec-binary/genvalidity-hspec-binary.cabal]: MIT

File: genvalidity-hspec-binary/genvalidity-hspec-binary.cabal [hunk 3]
```diff
@@ -0,0 +1,41 @@
+name:                genvalidity-hspec-binary
+version:             0.1.0.0
```
Comment (NorfairKing) [inline genvalidity-hspec-binary/genvalidity-hspec-binary.cabal]: 0.0.0.0

File: genvalidity-hspec-binary/genvalidity-hspec-binary.cabal [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+-- synopsis:
```
Comment (NorfairKing) [inline genvalidity-hspec-binary/genvalidity-hspec-binary.cabal]: Please don't leave these blank. I can't upload the package without it.

File: genvalidity-hspec-binary/genvalidity-hspec-binary.cabal [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+-- description:
+homepage:            https://github.com/githubuser/genvalidity-hspec-binary#readme
```
Comment (NorfairKing) [inline genvalidity-hspec-binary/genvalidity-hspec-binary.cabal]: Just point to the validity repo, please.

File: genvalidity-hspec-binary/genvalidity-hspec-binary.cabal [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+license:             BSD3
+license-file:        LICENSE
+author:              Author name here
```
Comment (NorfairKing) [inline genvalidity-hspec-binary/genvalidity-hspec-binary.cabal]: Make yourself the author

File: genvalidity-hspec-binary/genvalidity-hspec-binary.cabal [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+maintainer:          example@example.com
+copyright:           2017 Author name here
```
Comment (NorfairKing) [inline genvalidity-hspec-binary/genvalidity-hspec-binary.cabal]: Make sure the copyright belongs to me.

File: genvalidity-hspec-binary/genvalidity-hspec-binary.cabal [hunk 8]
```diff
@@ -0,0 +1,41 @@
+name:                genvalidity-hspec-binary
+version:             0.1.0.0
+-- synopsis:
+-- description:
+homepage:            https://github.com/githubuser/genvalidity-hspec-binary#readme
+license:             BSD3
+license-file:        LICENSE
+author:              Author name here
+maintainer:          example@example.com
```
Comment (NorfairKing) [inline genvalidity-hspec-binary/genvalidity-hspec-binary.cabal]: Make me the maintainer. See the other cabal files.

File: genvalidity-hspec-binary/genvalidity-hspec-binary.cabal [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+copyright:           2017 Author name here
+category:            Web
```
Comment (NorfairKing) [inline genvalidity-hspec-binary/genvalidity-hspec-binary.cabal]: Make this 'Testing'.

File: genvalidity-hspec-binary/genvalidity-hspec-binary.cabal [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+extra-source-files:  README.md
+cabal-version:       >=1.10
+
+library
+  hs-source-dirs:      src
+  exposed-modules:     Lib
+  build-depends:       base >= 4.7 && < 5
+  default-language:    Haskell2010
+
+executable genvalidity-hspec-binary-exe
```
Comment (NorfairKing) [inline genvalidity-hspec-binary/genvalidity-hspec-binary.cabal]: No executable, please remove this section.

File: genvalidity-hspec-binary/genvalidity-hspec-binary.cabal [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+maintainer:          example@example.com
+copyright:           2017 Author name here
+category:            Web
+build-type:          Simple
+extra-source-files:  README.md
+cabal-version:       >=1.10
+
+library
+  hs-source-dirs:      src
+  exposed-modules:     Lib
```
Comment (NorfairKing) [inline genvalidity-hspec-binary/genvalidity-hspec-binary.cabal]: That's definitely not accurate.

File: genvalidity-hspec-binary/genvalidity-hspec-binary.cabal [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+
+executable genvalidity-hspec-binary-exe
+  hs-source-dirs:      app
+  main-is:             Main.hs
+  ghc-options:         -threaded -rtsopts -with-rtsopts=-N
+  build-depends:       base
+                     , genvalidity-hspec-binary
+  default-language:    Haskell2010
+
+test-suite genvalidity-hspec-binary-test
```
Comment (NorfairKing) [inline genvalidity-hspec-binary/genvalidity-hspec-binary.cabal]: You need two test suites. One for the doctests, and one for the spec's.

**Comment by Nickske666** [comment]
I adapted everything except I don't understand what you want me to do with the .travis.yml file?





________________________________

**Pull request merged**